NEWS

```diff
@@ -7,7 +7,46 @@
 ##
 #
 
-## 1.3.0 (February 15, 2018)
+## Current
+
+## 1.4.0-rc1(September 7, 2018)
+
+Features:
+- Improved support for installation with latest ROCm
+- Improved support for latest rdma-core
+- Adding support for CUDA IPC for inter node GPU
+- Added support for latest Mellanox devices
+
+Bugfixes:
+- Performance fixes for rendezvous' protocol
```

Comment:
which one is it?

Reply:
I think it was about alignment

Reply:
not sure what you mean here, let's remove

Reply:
https://github.com/openucx/ucx/pull/2830